aptos-move/framework/aptos-stdlib/sources/table.move

```diff
@@ -9,7 +9,7 @@ module aptos_std::table {
     friend aptos_std::table_with_length;
 
     /// Type of tables
-    struct Table<phantom K: copy + drop, phantom V> has store {
+    struct Table<phantom K, phantom V> has store {
```

Comment:
Sometimes, the constraint is a way to help users. is there any scenario that copy or drop not required? 
For example, if I accidentally use a struct (for example, token) that is not droppable as a key, I later maynot be able to remove the item from the table since I cannot destroy it.

Reply:
The copy/drop constraints are already enforced on creation so technically even if the struct def doesn't enforce it, it's not possible to create a table with non-copyable non-droppable keys.

Reply:
Ok, we still need these constraints. Only ability at the struct level is redundant since the function already enforce these constraints.

Reply:
From Rust perspective, it doesn't have to be copy + drop. Even if we don't enforce on creation, users will find it is infeasible to use in their case by the api we provide so it won't be a technical issue.